apps/api/src/.env.test

```
@@ -42,6 +42,7 @@ S3_ACCESS_KEY=
 S3_SECRET=
 REDIS_ARENA_PORT=4568
 FRONT_BASE_URL='^https?://([a-zA-Z-_0-9]+.)?(dashboard(-v[0-9]+)?.novu(-staging)?.co)$'
+DASHBOARD_URL=http://127.0.0.1:4201
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DASHBOARD_URL key should go before the DISABLE_USER_REGISTRATION key

(UnorderedKey)